flink-rpc/flink-rpc-akka/pom.xml

```diff
@@ -69,6 +69,11 @@ under the License.
 			<artifactId>scala-compiler</artifactId>
 			<scope>compile</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.scala-lang.modules</groupId>
+			<artifactId>scala-parser-combinators_${scala.binary.version}</artifactId>
```

Comment:
please move this down into the `<dependencyManagement>` section; that way we don't change the structure of the dependency tree, but just modify the version. (i.e., the dependency tree still shows where this dependency came from)

Reply:
> 
> 
> please move this down into the `<dependencyManagement>` section; that way we don't change the structure of the dependency tree, but just modify the version. (i.e., the dependency tree still shows where this dependency came from)

Done!